### PR TITLE
switch to easybridge cla

### LINKED ISSUE #309: Adopt new Communitybridge/Linux Foundation CLA bot
https://lfcla.com/ is the new Linux Foundation CLA bot, and we should switch to using it as it's less cumbersome than the existing Linux Foundation Account requirement.

### PR DESCRIPTION
**Pull request by lizthegrey**
Fixes #309

### CODE REVIEW

File: docs/how-to-configure-new-repository.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ policies:
    repositores may opt-in for bigger number of required reviews.
 3. Every repo MUST enforce `cla/linuxfoundation` check for `Require status
    checks to pass before merging` section.
+   During the transition period, the CNCF `EasyCLA` check MAY be enabled.
```
Comment (yurishkuro) [inline docs/how-to-configure-new-repository.md:51]: should the wording above "MUST enforce `cla/linuxfoundation` check" still remain?

Comment (lizthegrey) [reply docs/how-to-configure-new-repository.md:51]: I can say `MAY be enabled instead of cla/linuxfoundation`, does that work?

Comment (yurishkuro) [reply docs/how-to-configure-new-repository.md:51]: Since this is talking about configuring new repos, do we even need to mention the old CLA?

Comment (lizthegrey) [reply docs/how-to-configure-new-repository.md:51]: No, because I think it also reflects our policy for existing repos, de facto.

Comment (yurishkuro) [reply docs/how-to-configure-new-repository.md:51]: oh... the file is `docs/how-to-configure-new-repository.md`

in this case I would phrase it "at least one of these CLA checks MUST be required"

Comment (lizthegrey) [reply docs/how-to-configure-new-repository.md]: PTAL

**Pull request merged**